src/components/ReservationForm.tsx

```diff
@@ -0,0 +1,417 @@
+          onClick={(): void =>
+            dispatch({ type: CalendarAction.CloseReservationForm })
+          }
+        >
+          <CloseIcon />
+        </IconButton>
+        <Typography variant="h6">Make Reservation</Typography>
+      </Toolbar>
+
+      {!isSubmitionCompleted && (
+        <React.Fragment>
+          <DialogContent>
+            <Formik
+              initialValues={{
+                event: state.currentEvent,
+                currentCategory: "",
+                searchString: "",
+                phone: "",
+                description: "",
+                guests: "",
+                project: projects[0],
+                liveRoom: "no",
+                hasGuests: "no",
+                hasNotes: "no",
+                hasEquipment: "no",
+                group: groups.filter(function (group) {return group.projectId === projects[0].id;})[0],
+                equipment: quantities,
+                filters,
+              }}
+              onSubmit={(values, { setSubmitting }): void => {
+                setSubmitting(true);
+
+                // sets the project property of values here because it
+                // doesn't update fast enough to set in the handleChange
+                setTimeout(() => {
+                  console.log(values);
+                  setSubmitionCompleted(true);
+                }, 2000);
+              }}
+              validationSchema={validationSchema}
+            >
+              {(props): any => {
+                const {
+                  values,
+                  touched,
+                  errors,
+                  isSubmitting,
+                  handleChange,
+                  setFieldValue,
+                  handleBlur,
+                  handleSubmit,
+                } = props;
+                return (
+                  <form onSubmit={handleSubmit}>
+                    <div className={classes.list}>
+                      <div style={{ paddingTop: 16 }}>Project:</div>
+                      <div style={{ paddingLeft: 5 }}>
+                        <Select
+                          dispatch={dispatch}
+                          state={state}
+                          value={values.project.id}
+                          selectName="projects"
+                          selectId="projectsDropDown"
+                          contents={projects}
+                          onChange={(event): void => {
+                            setFieldValue("project", findProjectById(projects,event?.target.value));
+                          }}
+                        ></Select>
+                      </div>
+                    </div>
+                    <div className={classes.list}>
+                      Group:{" "}
+                      <div style={{ paddingLeft: 10 }}>
+                        {groups
+                          .filter(function (group) {
+                            return group.projectId === values.project.id;
+                          })[0]
```

Comment:
arrow much easier to read
```suggestion
                        {groups
                          .filter((group) => group.projectId === values.project.id)[0]
```